apps/web/tsconfig.json

```diff
@@ -10,7 +10,8 @@
       "$/*": ["./src/*"],
       "$core/*": ["../../packages/core/src/*"],
       "$jobs/*": ["../../packages/jobs/src/*"],
-      "$ui/*": ["../../packages/web-ui/src/*"]
+      "$ui/*": ["../../packages/web-ui/src/*"],
+      "$compiler/*": ["../../packages/compiler/src/*"]
```

Comment:
Is compiler used by web? Or is used by core? If it's used by core put this in core tsconfig.json

Reply:
It's used by web-ui, and will be used by core too in the future.

Reply:
But does it web need to know about it here?

Reply:
Yes, in the same way that `web` needs to install all `web-ui` dependencies too. Since it is importing the original `src/index.ts` instead of a dist build, it's the parent package who needs to resolve all those imports; in this case, the `web` package.

Reply:
Can you try this in web-ui/tsconfig.json for compiler?
https://github.com/latitude-dev/latitude-llm/pull/34/files#r1677555026

Maybe that way this relation is not necesary

`web -> web-ui -> compiler`. Web and compiler shouldn't know about each other

Reply:
Does not work. Web still needs to resolve `$compiler`